wagtail/embeds/tests/test_embeds.py

```diff
@@ -652,7 +652,7 @@ def test_instagram_oembed_only_accepts_new_url_patterns(self):
     def test_instagram_oembed_return_values(self, urlopen):
         urlopen.return_value = self.dummy_response
         result = InstagramOEmbedFinder(app_id="123", app_secret="abc").find_embed(
-            "https://instagr.am/p/CHeRxmnDSYe/"
+            "https://instagram.com/p/CHeRxmnDSYe/"
```

Comment:
For future reference: these changes to the test are necessary because the new version of `InstagramOEmbedFinder` does a regexp match on the URL during `find_embed` to find the correct endpoint, whereas the old version only did that during `accept`, and used a constant endpoint in `find_embed`. `instagr.am` URLs do not match the regexp, so when testing `find_embed` in isolation, this now fails. However, in real-world usage `instagr.am` URLs would have been rejected at the `accept` step anyway - as confirmed by `test_instagram_oembed_only_accepts_new_url_patterns` - so this is testing an input that would not occur in reality, and there is no change in user-facing behaviour.